config-vm.md

```diff
@@ -0,0 +1,49 @@
+# <a name="VirtualMachineSpecificContainerConfiguration" /> Virtual-machine-specific Container Configuration
+
+Virtual-machine-based containers require additional configuration to that specified in the [default spec configuration](config.md).
+
+This **optional** configuration is specified in a "VM" object:
+
+* **`imagePath`** (string, required) host path to file that represents the root filesystem for the container virtual machine.
```

Comment:
“required” → “REQUIRED” (and similarly below, also “optional” → “OPTIONAL”.  The casing needs to match the [declarations here][1].

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/spec.md#notational-conventions

Reply:
Done.